timer/src/test/java/com/toggl/timer/startedit/domain/ProjectTagChipSelectorTests.kt

```diff
@@ -29,11 +32,11 @@ internal class ProjectTagChipSelectorTests {
 
         val chips = projectTagChipSelector(projectLabel, tagLabel, editableTimeEntry, projects, tags)
 
-        chips.shouldContain(ChipViewModel.AddProject(projectLabel))
+        chips.filterIsInstance<ChipViewModel.AddProject>().singleOrNull { it.text == projectLabel } shouldNotBe null
```

Comment:
```suggestion
        chips.filterIsInstance<ChipViewModel.AddProject>().singleOrNull { it.text == projectLabel }.shouldNotBeNull()
```

These are extreme 💄 suggestions. Feel free to ignore them, I just thought that it's more consistent with the `.shouldBeEmpty()` you already used below.